webrtc/RTCPeerConnection-createDataChannel.html

```diff
@@ -3,44 +3,51 @@
 <title>RTCPeerConnection.prototype.createDataChannel</title>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
+<script src="RTCPeerConnection-helper.js"></script>
 <script>
 'use strict';
 
-// Test is based on the following editor draft:
-// https://rawgit.com/w3c/webrtc-pc/cc8d80f455b86c8041d63bceb8b457f45c72aa89/webrtc.html
+// Test is based on the following revision:
+// https://rawgit.com/w3c/webrtc-pc/1cc5bfc3ff18741033d804c4a71f7891242fb5b3/webrtc.html
 
 /*
   6.1.  RTCPeerConnection Interface Extensions
 
     partial interface RTCPeerConnection {
-      RTCDataChannel createDataChannel(USVString label,
-                                        optional RTCDataChannelInit dataChannelDict);
-      ...
+        [...]
+        RTCDataChannel createDataChannel(USVString label,
+                                         optional RTCDataChannelInit dataChannelDict);
+        [...]
     };
 
   6.2.  RTCDataChannel
 
     interface RTCDataChannel : EventTarget {
-       readonly attribute USVString           label;
-       readonly attribute boolean             ordered;
-       readonly attribute unsigned short?     maxPacketLifeTime;
-       readonly attribute unsigned short?     maxRetransmits;
-       readonly attribute USVString           protocol;
-       readonly attribute boolean             negotiated;
-       readonly attribute unsigned short?     id;
-       readonly attribute RTCPriorityType     priority;
-       readonly attribute RTCDataChannelState readyState;
+        readonly attribute USVString           label;
+        readonly attribute boolean             ordered;
+        readonly attribute unsigned short?     maxPacketLifeTime;
+        readonly attribute unsigned short?     maxRetransmits;
+        readonly attribute USVString           protocol;
+        readonly attribute boolean             negotiated;
+        readonly attribute unsigned short?     id;
+        readonly attribute RTCPriorityType     priority;
+        readonly attribute RTCDataChannelState readyState;
+        readonly attribute unsigned long       bufferedAmount;
+                 attribute unsigned long       bufferedAmountLowThreshold;
+        [...]
+                 attribute DOMString           binaryType;
+        [...]
     };
 
     dictionary RTCDataChannelInit {
-      boolean         ordered = true;
-      unsigned short  maxPacketLifeTime;
-      unsigned short  maxRetransmits;
-      USVString       protocol = "";
-      boolean         negotiated = false;
-      [EnforceRange]
-      unsigned short  id;
-      RTCPriorityType priority = "low";
+        boolean         ordered = true;
+        unsigned short  maxPacketLifeTime;
+        unsigned short  maxRetransmits;
+        USVString       protocol = "";
+        boolean         negotiated = false;
+        [EnforceRange]
+        unsigned short  id;
+        RTCPriorityType priority = "low";
```

Comment:
Ditto for these, we should try to match the spec.
I would prefer removing these.